maskrcnn_benchmark/config/paths_catalog.py

```diff
@@ -2,11 +2,10 @@
 """Centralized catalog of paths."""
 
 import os
-
+from . import cfg
```

Comment:
If we don't have the `FORCE_USE_JSON_ANNOTATION`, this import won't be necessary